waku/node/waku_node.nim

```diff
@@ -389,14 +387,45 @@ proc filterSubscribe*(node: WakuNode, pubsubTopic: PubsubTopic, contentTopics: C
 
       await handler(pubsubTopic, message)
 
```

Comment:
If `pubsubTopic.isSome()` we want to skip the unnecessary iteration and autoshard considerations and simply publish the filter request as it is.

Reply:
true